pyscf/dft/libxc.py

```diff
@@ -770,7 +770,7 @@ def _xc_key_without_underscore(xc_keys):
 # Issue 1480
 if not hasattr(__config__, 'B3LYP_WITH_VWN5'):
     warnings.warn('Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, '
-                  'the same to the B3LYP functional in Gaussian and ORCA (issue 1480). '
+                  'the same to the B3LYP functional in Gaussian (issue 1480). '
                   'To restore the VWN5 definition, you can put the setting '
                   '"B3LYP_WITH_VWN5 = True" in pyscf_conf.py')
```

Comment:
What is the point of that keyword? Just use B3LYP5 if you want B3LYP5.

Reply:
Do you think something should be modified here? That looks introduced in #1481.

Reply:
Maybe @sunqm  can elucidate the introduction of that keyword? Would not a simple instruction to use the `B3LYP5` functional, instead, suffice?

Reply:
This keyword allows XC parser working the same to old versions.